dev/loadgen/pkg/loadgen/executor.go

```diff
@@ -112,7 +117,7 @@ type WsmanExecutor struct {
 
 // StartWorkspace starts a new workspace
 func (w *WsmanExecutor) StartWorkspace(spec *StartWorkspaceSpec) (callDuration time.Duration, err error) {
-	ctx, cancel := context.WithTimeout(context.Background(), 120*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 240*time.Second)
```

Comment:
@aledbf wdyt of this timeout? I assume it matches your expectation, but wanted to double check.